Detectors/TPC/simulation/include/TPCSimulation/Detector.h

```diff
@@ -28,40 +28,70 @@ namespace TPC {
 class Detector: public o2::Base::DetImpl<Detector> {
 
   public:
-    /**      Name :  Detector Name
-     *       Active: kTRUE for active detectors (ProcessHits() will be called)
-     *               kFALSE for inactive detectors
+   /** Local material/media IDs for TPC */
+   enum EMaterial {
```

Comment:
preferably scoped enum

Reply:
In case of a scoped enum at this point we need to change the interface of the MaterialManager and add an explicit type conversion to int before forwarding it to the coresponding VMC. Also, all detector/module class implementations should to be adapted accodingly.
@sawenzel Can/should we do that?